### PR TITLE
[DUOS-2174][DUOS-2347][risk=no] Remove v1 match algorithm

### PR DESCRIPTION
**Pull request by rushtong**
## Addresses
Partially addresses https://broadworkbench.atlassian.net/browse/DUOS-2174
Fully addresses https://broadworkbench.atlassian.net/browse/DUOS-2347 which is a partial duplicate of 2174

This PR removes the v1 match algorithm that has been unused since Consent migrated to using the v2 match endpoint.
Future PRs will remove other deprecated code paths that make use of `UseRestriction`.

----
Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/test/java/org/broadinstitute/dsde/consent/ontology/DataUseOwlTest.java [hunk 1]
```diff
@@ -41,36 +32,36 @@ public static void after() {
     @Test
     public void testNegativeMethodsAgainstInverse() throws Exception {
         UseRestriction consent = new Not(new Named(METHODS_RESEARCH));
-        Boolean b = ONT_MODEL_CACHE.matchPurpose(new MatchMessage(resources, new MatchPair(methodsPurpose, consent)));
-        assertFalse(b);
+//        Boolean b = ONT_MODEL_CACHE.matchPurpose(new MatchMessage(resources, new MatchPair(methodsPurpose, consent)));
+//        assertFalse(b);
```
Comment (fboulnois) [inline src/test/java/org/broadinstitute/dsde/consent/ontology/DataUseOwlTest.java]: Should these tests be removed?

**Review by fboulnois — COMMENTED** [review]
lgtm, one question:

File: src/test/java/org/broadinstitute/dsde/consent/ontology/DataUseOwlTest.java (hunk 1 above)
Comment (rushtong) [reply src/test/java/org/broadinstitute/dsde/consent/ontology/DataUseOwlTest.java]: Yup ... I missed that, thank you!

**Review by fboulnois — APPROVED** [review]

**Review by nmalfroy — APPROVED** [review]
lgtm!

**Comment by rushtong** [comment]
@nmalfroy @fboulnois  - very sorry but I missed a bunch of classes in the scala `automation` directory. These tests are run each morning (I tested manually) and report results to QA's BQ tables. We're in the middle of migrating all of those to java based tests.

**Pull request merged**